repository/project.go

```diff
@@ -144,7 +144,10 @@ func (r *ProjectRepository) UpdateProject(ctx context.Context, projectID uuid.UU
 		"name":                      p.Name,
 		"slackChannelID":            p.SlackChannelID,
 		"releaseNotificationConfig": model.ReleaseNotificationConfig(p.ReleaseNotificationConfig), // converted to the struct with json tags (the field is saved as json in the database)
-		"githubRepositoryURL":       p.GithubRepositoryURL.String(),
+		"githubRepositoryURL":       p.GithubRepositoryURL.String(),                               // TODO remove
+		"githubOwnerSlug":           p.GetGithubOwnerSlug(),
+		"githubRepoSlug":            p.GetGithubRepoSlug(),
+		"githubRepoURL":             p.GetGithubRepoURL().String(),
```

Comment:
Because Project has pointer to GithubRepo 

```
type Project {
....
GithubRepo *GithubRepo
}
```

if i would access GithubRepo fields directly, I would risk panic


```

var p svcmodel.Project

p.GithubRepo.OwnerSlug <-- possible panic (null pointer)
```

Therefore I added Getters